docker/platform/entrypoint.sh

```diff
@@ -22,7 +22,22 @@ touch /tmp/results/disk_bandwidth.log
 chown 1000:1000 /tmp/results/disk_bandwidth.log
 iotop -o -P -b >& /tmp/results/disk_bandwidth.log &
 
-echo "Starting xeon pcm-power collection"
+is_xeon=`lscpu | grep -i xeon | wc -l`
+
+if [ "$is_xeon"  == "1"  ]
```

Comment:
will this only return 1? for core, my machine returns '3', do we want to change the condition to '> 0' ?